src/three-components/ModelScene.js

```diff
@@ -177,6 +177,11 @@ export default class ModelScene extends Scene {
       // In practice, invocations of setSize are throttled at the element level,
       // so no need to throttle here:
       this.updateFraming();
+      // Render a frame right away
+      this.isDirty = true;
+      this.camera.aspect = this.aspect;
+      this.camera.updateProjectionMatrix();
+      this.renderer.render(performance.now());
```

Comment:
When the camera's aspect ratio changes, that changes the framing of the scene, which is done by changing other camera parameters. Perhaps the render call should be triggered by the [$onResize] method in controls.ts instead, since that takes care of updating the camera.

Reply:
Feel free to take over the PR.
I'm unable to maintain three.js and work on <model-viewer> at the same time.

Reply:
Sure thing, understood. Thanks for the help!

Reply:
Are those glitches you made public? I can't seem to access them, and I'd like to update them with my own fix to ensure it works. I'm still new to glitch, so it's helpful to have something to work from. Thanks!

Reply:
I just made the `model-viewer-tests` glitch public 👌
https://glitch.com/~model-viewer-tests

Reply:
Thank you!